src/assets/styles/_kongponents-overrides.scss

```diff
@@ -8,6 +8,13 @@
   align-items: center;
 }
 
+.kong-card .k-card-body {
+  // Overrides some properties that really shouldn’t be controlled by the KCard component.
+  color: currentColor !important;
+  font-size: inherit !important;
+  line-height: inherit !important;
+}
+
```

Comment:
I’m using `inherit` here so that this element inherits the font size and line height from parent elements which has the effect that it uses the default font sizes _we use in the project_ (i.e. not browser defaults). The same applies to `currentColor` and in this case `inherit`.

With the default styles of the card applied, the font size of card content is set to 13 pixels by default and in most cases, that looks too small to me and breaks out of the general styles of the project. Generally, it shouldn’t be a component library being opinionated about how the base styles of a project look. This can be part of a design system or a project’s custom definitions, but should be kept out of components who should just adapt to such defaults.

The difference is easier observed by going to a page with a KCard component, selecting the element with the `k-card-body` class, and toggling the three override property declarations in the Styles panel.

---

Example: https://deploy-preview-754--kuma-gui.netlify.app/gui/zoneegresses?zoneEgress=zoneegress-1&tab=insights

**Currently**:

![Screenshot from 2023-03-24 09-50-47](https://user-images.githubusercontent.com/5774638/227471028-de6540e2-8c40-4abd-b48c-f5fa42382eec.png)

**In this PR**:

![Screenshot from 2023-03-24 09-50-55](https://user-images.githubusercontent.com/5774638/227471020-074f8698-08b1-4e23-9fbd-bfb31d80732e.png)

---

Example: https://deploy-preview-754--kuma-gui.netlify.app/gui/

**Currently**:

![Screenshot from 2023-03-24 09-57-43](https://user-images.githubusercontent.com/5774638/227472487-2d4c1962-19c3-4946-81fe-30d8fd1f51af.png)

**In this PR**:

![Screenshot from 2023-03-24 09-57-35](https://user-images.githubusercontent.com/5774638/227472494-9235864f-80ba-4eef-b720-61fe5912da3d.png)

Reply:
ty! I'm just reading over/looking at this in more detail.

Just while I'm doing that I wanted to ask about the first set of screengrabs where the second level headers are now bigger than the first level headers, here:

<img width="333" alt="Screenshot 2023-03-24 at 10 24 25" src="https://user-images.githubusercontent.com/554604/227496203-6efc0155-efeb-422f-aa73-3f8af4d34047.png">

Are we ok with this?

Reply:
True, that is not good.

The heading hierarchy is broken in this view for starters. I use the following naive script to get a document’s heading hierarchy:

```js
console.log(Array.from(document.querySelectorAll('h1,h2,h3,h4,h5,h6')).map((h) => ('-'.repeat(parseInt(h.tagName[1]))) + ` ${h.textContent.trim()}`).join('\n'))
```

Note how we skip levels 2 and 3 entirely here.

```
- Zone Egress: zoneegress-1
---- Connect time: July 13, 2021 at 10:41:04 AM
----- General Information:
------ cds:
------ eds:
------ lds:
------ rds:
```

If I fix this and remove the font size override from the too small headings here (e.g. General Information:):

```
- Zone Egress: zoneegress-1
-- Connect time: July 13, 2021 at 10:41:04 AM
--- General Information:
---- cds:
---- eds:
---- lds:
```

![image](https://user-images.githubusercontent.com/5774638/227499189-d11e3cd1-751b-4d12-8237-20c4034a1d1e.png)

(I also fixed the issue of sometimes showing a heading when we don’t have a value to display).